src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -1061,9 +1061,9 @@ Conditions, violations, incidents, and issues all work for any entity.
     id="cust-evnts"
     title="Custom events"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic 🟡
 
-    OpenTelemetry users can report custom events by using New Relic’s [Event API](/docs/data-apis/ingest-apis/introduction-event-api/).
+    You can report custom events from OpenTelemetry by using the New Relic [Event API](/docs/data-apis/ingest-apis/introduction-event-api/). Alternatively, you can report on custom events using the OpenTelemetry events API via OTLP. These events are currently stored and queried as New Relic logs.
```

Comment:
consider adding link to logs section of data types doc where we mention 'New Relic logs'. https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#log-data

Reply:
@zuluecho9 I added a link to data types:

These events are currently stored and queried as New Relic logs. For more about logs, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/#log-data).